requirements.txt

```diff
@@ -2,3 +2,4 @@ click
 thoth-common
 pandas
 pygit2
+fasttext
```

Comment:
You can add this also to Pipfile/Pipfile.lock so we can do recommendations on this stack.

Reply:
Updated